docs/source/distributions/importing_as_library.md

```diff
@@ -17,7 +17,7 @@ client = LlamaStackAsLibraryClient(
     # provider_data is optional, but if you need to pass in any provider specific data, you can do so here.
     provider_data={"tavily_search_api_key": os.environ["TAVILY_SEARCH_API_KEY"]},
 )
-client.initialize()
+# Client is automatically initialized and ready to use
```

Comment:
coding agents are great at adding info that's no longer relevant

Reply:
removed 👍🏽